src/media/js/addon/components/version.js

```diff
@@ -63,21 +64,14 @@ export default class AddonVersion extends React.Component {
 
         {this.props.showReviewActions &&
          this.props.status !== constants.STATUS_OBSOLETE &&
-          <div className="addon-version-review-actions">
-            {this.props.status !== constants.STATUS_REJECTED &&
-              <button onClick={this.reject} disabled={reviewActionsDisabled}>
-                {this.props.isRejecting ? 'Rejecting...' : 'Reject'}
-              </button>
-            }
-            {this.props.status !== constants.STATUS_PUBLIC &&
-              <button onClick={this.publish} disabled={reviewActionsDisabled}>
-                {this.props.isPublishing ? 'Publishing...' : 'Publish'}
-              </button>
-            }
-          </div>
+          <ReviewActions isProcessing={this.props.isPublishing ||
+                                       this.props.isRejecting}
+                         publish={this.publish}
+                         reject={this.reject}
+                         status={this.props.status}/>
         }
 
-        {this.props.notes.length &&
+        {this.props.notes && this.props.notes.length &&
```

Comment:
for important content of the component, i do like to see it in the main render method so that i don't have to jump up and down. better ergonomics on the code